.jenkinsci-new/text-variables.groovy

```diff
@@ -0,0 +1,212 @@
+param_chose_opt = 'Default\nBranch commit\nOn open PR\nCommit in Open PR\nBefore merge to trunk\nBefore merge develop\nBefore merge master\nNightly build\nCustom command'
+
+param_descriptions = """
+<p>
+  <strong>Default</strong> - will automatically chose the correct one based on branch name and build number<br />
+  <strong>Branch commit</strong> - Linux/gcc v5; Test: Smoke, Unit;<br />
+  <strong>On open PR -</strong> Linux/gcc v5, MacOS/appleclang; Test: Smoke, Unit; Coverage; Analysis: cppcheck, sonar;<br />
+  <strong>Commit in Open PR</strong> - Same as Branch commit<br />
+  <strong>Before merge to trunk</strong> - Linux/gcc v5 v7, Linux/clang v6 v7, MacOS/appleclang; Test: ALL; Coverage; Analysis: cppcheck, sonar; Build type: Debug when Release<br />
+  <strong>Before merge develop</strong> - Not implemented<br />
+  <strong>Before merge master</strong> - Not implemented<br />
+  <strong>Nightly build</strong> - Not implemented<br />
+  <strong>Custom command</strong> - enter command below, Ex: build_type='Release'; testing=false;<br />
+</p>
+"""
+
+cmd_description = """
+<h3>List of parameters for Jenkins "Custom command" option:</h3>
```

Comment:
can you apply formatting to HTML chunk for a better readability? tabs or spaces — whatever to understand what's inside